OSRMTextInstructions.podspec

```diff
@@ -0,0 +1,47 @@
+Pod::Spec.new do |s|
+
+  # ―――  Spec Metadata  ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.name = "OSRMTextInstructions.swift"
```

Comment:
In that case, I think we should rename the repo as well, but then it'd only be naming conventions that would differentiate the two libs.